client/src/app/api/login/route.ts

```diff
@@ -18,10 +18,10 @@ export async function POST(request: NextRequest) {
   try {
     const body = await request.json();
     const { email, password } = body;
-    const response = await login(email, password);
-    await setCookie(CookieType.ACCESS_TOKEN, response.token);
-    await setCookie(CookieType.USER, JSON.stringify(response.user));
-    return response;
+    const loginResponse = await login(email, password);
+    setCookie(CookieType.ACCESS_TOKEN, loginResponse.token);
+    setCookie(CookieType.USER, JSON.stringify(loginResponse.user));
+    return NextResponse.json(loginResponse);
```

Comment:
tried to make this a `NextResponse.redirect(config.api.baseUrl)` but that didn't redirect